packages/protvista-variation-adapter/src/ProtvistaVariationAdapter.ts

```diff
@@ -1,38 +1,43 @@
 import ProtvistaFeatureAdapter from "protvista-feature-adapter";
 import { v1 } from "uuid";
+import { NightingaleElement } from "data-loader";
```

Comment:
There's no reason that this definition should be kept in `data-loader`, I guess it'd be good to have a common place not in any specific package with all of the common code and interface. Do you know if there's a way to do that in lerna? Or do we just import from a relative path up outside the packages themselves?